src/main/python/rlbot/setup_manager.py

```diff
@@ -242,6 +243,10 @@ def load_match_config(self, match_config: MatchConfig, bot_config_overrides={}):
         self.names = [bot.name for bot in match_config.player_configs]
         self.teams = [bot.team for bot in match_config.player_configs]
 
+        for player in match_config.player_configs:
+            if player.bot and not player.rlbot_controlled:
+                set_random_psyonix_bot_preset(player)
+
         bundles = [bot_config_overrides[index] if index in bot_config_overrides else
```

Comment:
This line seems to be designed for overriding configs which might work for those cased. Not sure if it has been used before.